modules/common/logging/hw-mac-retrieve.nix

```diff
@@ -28,15 +29,19 @@ in
     # TODO: Remove hw-mac.service and replace with givc rpc later
     systemd.services."hw-mac" = {
       description = "Retrieve MAC address from net-vm";
-      wantedBy = [ "alloy.service" ];
+      wantedBy =
+        if hostConfig.ghaf.virtualization.microvm.guivm.enable then
+          [ "multi-user.target" ]
+        else
+          [ "alloy.service" ];
```

Comment:
Is it possible to use `wantedBy = [ "multi-user.target" , "alloy.service" ];` and avoid passing hostConfig?

Reply:
I thought it wouldn't work but not tried before. Yes, use it like you mentioned is much easier solution as it fixed.